CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.6.0 - 2016-03-30
+
+* Supported configuration of `specs` for certain browsers (see [#9]).
+* Added option `--grep` for running tests (see [#15]).
```

Comment:
Added option `--grep` for running tests -> Added option `--grep` for selecting specific tests